src/utils/map.js

```diff
@@ -9,7 +9,7 @@ export function loadMap(element, mapOptions) {
   return loadModules(
     ['esri/Map', 'esri/views/MapView', 'esri/Graphic'],
     // NOTE: keep this current w/ the latest version of the JSAPI
-    { css: 'https://js.arcgis.com/4.10/esri/css/main.css' }
+    { css: 'https://js.arcgis.com/4.11/esri/css/main.css' }
```

Comment:
let's just use the new `css: true` so this stays evergreen

Reply:
then we can remove the `// NOTE: keep this current w/ the latest version of the JSAPI` comment above.

Reply:
Cool feature, good to know. Updatet